agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs

```diff
@@ -102,12 +165,16 @@ public EncodedMessage Encode(SendMessageParameters sendMessageParameters)
                 Recipients = sendMessageParameters.Recipients
             };
 
+            var value = sendMessageParameters.TechnicalMessageType == TechnicalMessageTypes.GpsInfo
```

Comment:
This would mean, that we are encoding EFDI values incorrect, my suggestion would be to implement a function on the technical message type that is able to decide whether the value should be encoded or not.

Reply:
Fixed